html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationProvisionings.js

```diff
@@ -302,6 +356,26 @@ export const pfConfigurationProvisioningFields = {
       ]
     }
   },
+  critical_issues_threshold: ({ options: { meta = {} } } = {}) => {
+    return {
+      label: i18n.t('Critical issues threshold'),
+      text: i18n.t('Raise the non compliance security event the number of critical issues is greater or equal than this. 0 deactivates it.'),
```

Comment:
I don't find this very clear. How about *The minimum number of critical issues a device needs to have before it gets isolated. 0 deactivates it.*